drivers/at/at.c

```diff
@@ -6,6 +6,37 @@
  * directory for more details.
  */
 
+/* A note on URCs, regardless of whether URC handling is enabled or not.
+ *
+ * Some DCEs (e.g. LTE modules from uBlox) define a grace period where URCs are
```

Comment:
DCE -> data circuit-terminating equipment aka modem

URC -> Unsolicited Result Code

could you somehow add this (eg full name on first use)